cmd/vault-server/startcmd/start.go

```diff
@@ -7,25 +7,42 @@ SPDX-License-Identifier: Apache-2.0
 package startcmd
```

Comment:
where is the unit test for stat.go

Reply:
will add them in the next PR https://github.com/trustbloc/edge-service/pull/574